tests/ordering.test.js

```diff
@@ -153,4 +153,39 @@ describe('Ordering', () => {
       storedValue,
     );
   });
+
+  // issue: https://github.com/simple-icons/simple-icons-website/issues/66
+  it('does not have a black-like color at the start', async () => {
+    const colors = [
+      '181717',
+      '1D1717',
+      '000000',
+      '512BD4',
+      '40AEF0',
+      '0094F5',
+      'FF0000',
+      '004088',
+      '0099E5',
+      'EF2D5E',
+      'FF9E0F',
+      '071D49',
+      '00A98F',
+      '191A1B',
+      '41454A',
+      'A100FF',
+      '26689A',
+      'A9225C',
+      '83B81A',
+      '0085CA',
+      '0B2C4A',
+    ];
+
+    const hexes = sortByColors(colors);
+    console.log(hexes);
+    // should not match black colors
+    await expect(hexes[0]).not.toMatch('181717');
+    await expect(hexes[1]).not.toMatch('1D1717');
+
+    await expect(hexes[hexes.length - 1]).toMatch('000000');
```

Comment:
If you disagree with the comment above, it's more expressive to use `toEqual` as you want to know if it equals, not if it matches.

```suggestion
    await expect(hexes[hexes.length - 1]).toEqual('000000');
```

Reply:
yes, toEqual is the way to go for the full array anyways. Since we are comparing the whole output, this assertion goes away as superfluous

Reply:
Done, with these suggestions. Very useful.